librad/src/git/fetch.rs

```diff
@@ -30,13 +30,26 @@ use crate::{
     peer::PeerId,
 };
 
+/// Upper bound of 1GB for fetching data. This upper bound is useful for when
+/// fetching from an unknown (and possibly untrusted) peer.
+pub const ONE_GB: usize = 1024 * 1024 * 1024;
+/// Upper bound of 1MB for fetching data. This upper bound is an estimate for
+/// limiting the fetch of the `rad/*` references.
+pub const ONE_MB: usize = 1024 * 1024;
```

Comment:
I think those numbers are too big -- the branches considered by `All` / `Peek` are supposed to be _**mostly**_ identical, so the resulting packfile should be a lot smaller than `ONE_MB * number_of_rad_id_branches` (or else, something's fishy).

Any chance we could bubble these up as runtime parameters?